src/components/Hero/Hero.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
The layout looks already very good. Real close to the design. Only the font-weight of the h1 tag needs to be changed to 500 and the button needs a bit more horizontal spacing. When you enter `r/javascript` in the input it's also not perfectly aligned, but that might change when you change the input into a string

![image](https://user-images.githubusercontent.com/9722792/77847850-60358080-71c0-11ea-861c-ae691ab3bf4e.png)